generators/swift/codegen/src/ast/Param.ts

```diff
@@ -0,0 +1,41 @@
+import { AstNode, Writer } from "@fern-api/generator-commons";
+import Swift from "../swift";
+
+export declare namespace Param {
+    interface Args {
+        title: string;
+        type: string; // TODO
+        defaultValue?: string; // TODO
+    }
+}
+
+export class Param extends AstNode {
+
+    public readonly title: string;
+    public readonly type: string;
```

Comment:
@mikemilla i don't think you are going to want to model `type` as a string -- you are going to want a way to model a swift type (primitive + custom class).

Reply:
In C# you probably saw `csharp.Type.int()` or `csharp.Type.class({ name: "MyClass", namespace: "MyNamespac" });`

This adds some type safety to the `type` field, making it easier for non swift folks to contribute to the generator. Open to other ideas for modeling type as well but wanted to share what works in C#!

Reply:
Yep, that was just a placeholder as I was building things out. It'll be some form of type.